test/SubdomainWithDCLControllerV2.spec.js

```diff
@@ -2059,6 +2054,24 @@ describe('DCL Names V2 with DCLControllerV2', function () {
         expect(currentResolver).to.be.equal(ZERO_ADDRESS)
       })
 
+      it('should transfer the fee from the caller to the fee collector', async function () {
+        const userBalance = await manaContract.balanceOf(user)
+        const feeCollectorBalance = await manaContract.balanceOf(feeCollector)
```

Comment:
expect current balance?
and expect user balance?

Reply:
added price validation